.github/workflows/test_dbt_models.yaml

```diff
@@ -22,25 +34,49 @@ jobs:
         with:
           role-to-assume: ${{ secrets.AWS_IAM_ROLE_TO_ASSUME_ARN }}
 
+      - name: Restore dbt state cache
+        id: cache
+        uses: ./.github/actions/restore_dbt_cache
+        with:
+          path: ${{ env.PROJECT_DIR }}/${{ env.STATE_DIR }}
+          key: ${{ env.CACHE_KEY }}
```

Comment:
I adapted this logic from the `build-and-test-dbt` workflow:

https://github.com/ccao-data/data-architecture/blob/ac90562f411757e53b74349f2eace76621f81f7d/.github/workflows/build_and_test_dbt.yaml#L35-L40